ontology/yaml/resources/HVAC/entity_types/CT.yaml

```diff
@@ -118,6 +118,73 @@ CT_SS_VSC_SWTC_SWISOVPM_RWISOVPM_SWISOVM_MWVPM:
   - SWISOVM
   - MWVPM
 
+CT_SS_PFSC:
+  description: "Cooling tower with pump control."
+  is_canonical: true
+  implements:
+  - CT
+  - SS
+  - PFSC
```

Comment:
Pumps are independent of the cooling tower, unless these are spray pumps integral to the cooling tower. If so, should be more specific with the field names.

Reply:
Changed to CT_SS_FSC